mmv1/templates/terraform/examples/data_pipeline_pipeline.tf.erb

```diff
@@ -0,0 +1,49 @@
+resource "google_service_account" "service_account" {
+  account_id   = "service-account-id"
+  display_name = "Service Account"
+}
```

Comment:
Looks like the tests are failing due to this resource not getting cleaned up.

Could you define this in a way that the number is able to be cleaned up : ie 
https://github.com/GoogleCloudPlatform/magic-modules/blob/67f9ba8865dd64d9510fcea603f7e4b1bcbdb14e/mmv1/products/beyondcorp/AppConnection.yaml#L62-L68
https://github.com/GoogleCloudPlatform/magic-modules/blob/67f9ba8865dd64d9510fcea603f7e4b1bcbdb14e/mmv1/templates/terraform/examples/beyondcorp_app_connection_full.tf.erb#L2